core/domain/story_jobs_one_off.py

```diff
@@ -94,3 +94,55 @@ def reduce(key, values):
                 sum(ast.literal_eval(v) for v in values))])
         else:
             yield (key, values)
+
+
+class UpdateStoryThumbnailSizeOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job to update the thumbnail_size_in_bytes in story models. """
+
+    _DELETED_KEY = 'story_deleted'
+    _ERROR_KEY = 'update_error'
+    _SUCCESS_KEY = 'thumbnail_size_updated'
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [story_models.StoryModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        story = story_fetchers.get_story_by_id(item.id)
+
+        try:
+            # We are not updating thumbnail_filename here, but using it call the
+            # update for story thumbnail_size_in_bytes.
+            # old_value and new_value are same here, because the update for
+            # thumbnail_size_in_bytes is called from within the code for
+            # updating thumbnail_filename in story_services.py file.
+            commit_cmds = [story_domain.StoryChange({
+                'cmd': story_domain.CMD_UPDATE_STORY_PROPERTY,
+                'property_name': (
+                    story_domain.STORY_PROPERTY_THUMBNAIL_FILENAME),
+                'new_value': story.thumbnail_filename,
+                'old_value': story.thumbnail_filename
+            })]
+
+            story_services.update_story(
```

Comment:
Can you verify that this does not change the last-updated timestamp of the story (in tests)?

Reply:
I ran the job for Topics locally, the last-updated timestamp is changed. 
This should be an expected result right ? Because we are modifying the storage model by adding new attribute ? 
WDYT ?

Reply:
SGTM -- but note, you'll need to do update_last_updated_time=False or something like that -- see other usages. Make sure there's also a backend test that verifies that the timestamp is not changed.